.test-infra/jenkins/job_PreCommit_Java.groovy

```diff
@@ -42,7 +42,7 @@ builder.build {
           pattern('**/build/reports/checkstyle/*.xml')
         }
         findBugs {
```

Comment:
Any reason not to switch to spot bugs here?

Jenkins plugin also has that spotBugs() symbol (see [1], #81)

[1] https://github.com/jenkinsci/warnings-ng-plugin/blob/master/SUPPORTED-FORMATS.md